.gitignore

```diff
@@ -1,2 +1,3 @@
 vendor
 .glide
+types/types.pb.go
```

Comment:
are we removing it from Git? but abci won't compile without it, no?

Reply:
I know.. but if I make changes to it, I will not be able to commit them!

Reply:
so what's the point of adding it to gitignore?

Reply:
so we dont accidentally check in changes.

Reply:
To prevent automatic commit in git commit -a or plain git add. It’s a best practice for all auto generated files so that you don’t automatically commit every pointless variation to the repo like a time stamp or white space

Reply:
hm.. ok